contrib/poll_stations.py

```diff
@@ -182,7 +181,7 @@ def print_station(self, station):
           self.update_threads[remote_test] = update_thread
     except socket.error as e:
       print(' |-- Connection Error: %s' % e)
-    print
+    print()
```

Comment:
`from __future__ import print_function`

Can you please test these files in PY2?

Reply:
Added from __future__ import print_function. I see "Waiting for stations..." when running from PY2 and PY3 virtualenv